src/content/contributing/style-guide/index.md

```diff
@@ -0,0 +1,188 @@
+
+### Consistency {#consistency}
+
+Many of the topics covered on [Ethereum.org](http://ethereum.org) are technically complex. To reduce confusion to the reader, terms should be used consistently. For example, don't cycle back-and-forth between proof-of-work and PoW at random.
+
+### Mainnet {#mainnet}
+
+When referring to the Ethereum Mainnet (i.e. not referring to a testnet) use the proper noun. Proper nouns help avoid confusion and build greater understanding.
+
+**Correct usage:**
+
+- Mainnet
+- Ethereum Mainnet
+
+**Incorrect usage:**
+
+- main net
+- mainnet
+- Main net
+- Ethereum mainnet
+
+### Proof-of-work {#proof-of-work}
+
+Proof-of-work should be capitalized at the beginning of a sentence or when the term is introduced for the first time on a page, in any other instance, all letters should be lower case. In either case, proof-of-work should be hyphenated between each word.
+
+**Correct usage:**
+
+- Proof-of-work
+- proof-of-work
+
+**Incorrect usage:**
+
+- Proof-of-Work
+- Proof of work
+- proof of work
+
+### Proof-of-stake {#proof-of-stake}
+
+The same rules we apply to proof-of-stake are applicable to proof-of-stake.
+
+**Correct usage:**
+
+- Proof-of-stake
+- proof-of-stake
+
+**Incorrect usage:**
+
+- Proof-of-Stake
+- Proof of stake
+- proof of stake
+
+### Use Active Voice {#use-active-voice}
+
+Sentences using active voice are more concise and efficient, making your writing more engaging and easier to comprehend.
+
+**Active voice sentence:** an actor acts on a target
+
+> _ "The man bought a car."_
+
+**Passive voice sentence:** a target acts on an actor
+
+> _ "The car was bought by a man."_
+
+[Read more on active voice](https://www.grammarly.com/blog/active-vs-passive-voice/)
+
+_This isn't an easy one, especially for non-native English speakers. If you aren't sure, don't worry. We'll help with this one._
+
+### Linking to internal pages {#internal-links}
+
+When linking to another page on Ethereum.org, use the relative path over the absolute path. This maintains consistency across different language versions of the site and ensures links work correctly in local development.
+
+```md
+<--- Good --->
+
+Read more about [smart contracts](/docs/developers/smart-contracts)
+
+<--- Bad --->
+
+Read more about [smart contracts](https://ethereum.org/en/docs/developers/smart-contracts)
+```
+
+### Linking to images {#images}
+
+When adding an image to a page, the image should be downloaded and placed in the same folder as the markdown file. You can reference the image like this:
+
+`![alt text for image](./image.png)`
+
+```md
+<--- Good --->
+
+![How to mint your NFT](./mintYourNFT.gif)
+
+<--- Bad --->
+
+![How to mint your NFT](https://cdn-images-1.medium.com/max/2000/0342fj_fsdfs.gif)
+```
+
+### Using Emojis {#images}
+
+Everyone loves emojis <Emoji text="🥰" size={1} /> To stanardize the appearence of all Emojis across browsers [Ethereum.org](http://ethereum.org) uses an `Emoji` component.
```

Comment:
```suggestion
Everyone loves emojis <Emoji text="🥰" size={1} /> To standardize the appearance of all Emojis across browsers [Ethereum.org](http://ethereum.org) uses an `Emoji` component.
```

Reply:
Wish I could chalk this up to intl but I just can't spell it seems.